modules/swagger-codegen/src/main/java/io/swagger/codegen/utils/ModelUtils.java

```diff
@@ -0,0 +1,29 @@
+package io.swagger.codegen.utils;
+
+import io.swagger.codegen.CodegenModel;
+
+import java.util.List;
+import java.util.Map;
+
+public class ModelUtils {
+    public static CodegenModel getModelByName(final String name, final Map<String, Object> models) {
```

Comment:
Agreed that the function name is pretty self explanatory but still we want a proper docstring for newly-added functions if it's not too much work to the contributor.